src/Form/Type/Procuration/ProcurationRequestType.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace AppBundle\Form\Type\Procuration;
+
+use AppBundle\Entity\ElectionRound;
+use AppBundle\Entity\ProcurationRequest;
+use AppBundle\Form\UnitedNationsCountryType;
+use AppBundle\Repository\ElectionRoundRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\IsTrue;
+
+class ProcurationRequestType extends AbstractProcurationType
```

Comment:
getParent() ?

Reply:
Nope we don't want the component to handle this inheritance, there is no gain, only a performance loose.